lib/slack_interface/request_handler.coffee

```diff
@@ -112,7 +112,7 @@ class SlackInterfaceRequestHandler
                   reply_data['text'] = "And we're back. Hope you closed that deal!"
 
               when 'list'
-                if @auth.args[0]?
+                if @auth.args[0]? && @auth.args[0] != '|'
```

Comment:
Probably switch these - start with `if @auth.args[0] == '|'`, and then `else if @auth.args[0]?`. Keeps it clean.